### ISSUE TITLE
Support for @font-face

### ISSUE THREAD

**Issue by mylescc** [jonkemp/inline-css#42]
At the moment when I run inline-css it seems to remove all @font-face queries from my html file. Is there some way that, like media queries, these can be ignored?

**Comment by elliottregan** [comment]
`@font-face` won't work in nearly every email client: https://www.campaignmonitor.com/blog/email-marketing/2010/03/does-font-face-work-in-email/

**Comment by mylescc** [comment]
fair enough! I'm guessing support for font-face isn't going to improve any time soon either.

**Comment by elliottregan** [comment]
Then I'd say it should be handled in the same way as outlined in this issue: https://github.com/jonkemp/inline-css/issues/31

**Comment by shrpne** [comment]
@elliottregan your link to font-face support article is outdated
Here is actual support list (which is much more bigger): https://www.campaignmonitor.com/resources/guides/web-fonts-in-email/

**Comment by elliottregan** [comment]
@shrpne Great! I've definitely changed my perspective since then. I don't think that this package should be the arbiter of what is valid CSS. it should just inline all CSS as best it can.

**Comment by jonkemp** [comment]
Juice seems to support this. You can try that. The reason I made this in the first place is that Juice was no longer being maintained. It is now. I am not really interested in developing new features at this time.

https://github.com/Automattic/juice

**Comment by overbyte** [comment]
note you can add another <style> tag and add a `data-embed` attribute to do this

**Comment by seyfer** [comment]
@jonkemp could you please make it clear in the project readme on top?
It is just a waste of time. I was choosing between inline-css one and Juice, have chosen inline-css as most recent and modern just to find out a day later, accidentally, in this thread, that inline-css is not in development anymore.
Please add it to the readme, it can save hours if not days for somebody.
